build_all/windows/build.cmd

```diff
@@ -97,6 +97,19 @@ rem no error checking
 
 pushd %build-root%\cmake\%CMAKE_DIR%
 
+echo ***checking msbuild***
+where /q msbuild
+IF ERRORLEVEL 1 (
+echo ***setting VC paths***
+    IF EXIST "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsMSBuildCmd.bat" call "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsMSBuildCmd.bat"
+)
+
+where msbuild
+IF ERRORLEVEL 1 (
+    echo [91mERROR: msbuild not found!!![0m 
```

Comment:
are these non-ascii characters?

Reply:
yes, this are ESC chars which will cause the text to print in RED.